js/about.js

```diff
@@ -4,7 +4,7 @@ document.addEventListener('DOMContentLoaded', function() {
 		aboutEntry.addEventListener('click', function (event) {
 			event.stopPropagation();
 			event.preventDefault();
-			OCP.Loader.loadScript('firstrunwizard', 'firstrunwizard.js').then(function () {
+			OCP.Loader.loadScript('firstrunwizard', 'firstrunwizard-main.js').then(function () {
```

Comment:
Why not use webpack import?

Reply:
i think we wanted to keep the loader script small since it is loaded on every page, but might be something worth to look into. I don't think that the size is to heavy just with the webpack bootstrapping code